app/src/main/java/com/infomaniak/mail/utils/NotificationUtils.kt

```diff
@@ -235,7 +234,7 @@ class NotificationUtils @Inject constructor(
 
             @Suppress("MissingPermission")
             notificationsByMailboxId[mailboxId]?.let { notifications ->
-                notificationManagerCompat.notify(notifications)
+                notificationManagerCompat.notify(notifications.toList())
```

Comment:
The crash was that `notificationsByMailboxId[mailboxId]` throws OOB.
How can this fix solve the OOB ? 🤔